packages/@divvi/mobile/src/images/WelcomeLogo.tsx

```diff
@@ -9,11 +10,16 @@ interface Props {
 }
 
 export default function WelcomeLogo({ width = 243, height = 64, style }: Props) {
+  const CustomWelcomeLogo = getAppConfig().themes?.default?.assets?.welcomeLogo
+  if (CustomWelcomeLogo) {
+    return <CustomWelcomeLogo width={width} height={height} style={style} />
+  }
+
   return (
-    <Svg width={width} height={height} viewBox={`0 0 243 64`} fill="none" style={style}>
+    <Svg width={width} height={height} viewBox={`0 0 168 58`} fill="none" style={style}>
```

Comment:
There's a risk we could display the logo with an incorrect ratio.
Should we just get rid of the `width` / `height` passed to `WelcomeLogo`?
Or do we have different callers using different sizes?

Reply:
yeah true, i can remove. i don't know why the width and height were specified